### ISSUE TITLE
size prop interferes with some components

### ISSUE THREAD

**Issue by christophemarois** [e2o/vue-element-query#5]
Because `size` is a very common property name, vue-element-query interferes with a lot of components that use this prop name  (e.g., used in the icon component of Buefy), when set globally with `Vue.use()`.

Maybe renaming it to something more unique would be a better idea? Or maybe allowing to customize the prop name in the `Vue.use()` initialization would be a better way, something like:

```javascript
Vue.use(VueElementQuery, {
  sizeProp: 'elementQuerySize'
})
```

The advantage of doing it like this is that it would easy to make the change backward-compatible. Would you accept a pull request for this?

EDIT: maybe an optional `prefix` initialization option (add a prefix to all injected instance props) would be an even better idea.

**Comment by e2o** [comment]
Yeah sure, I would implement the second option to not break the current implementations. Feel free to open a PR for this. Thanks for raising the issue :)

**Comment by e2o** [comment]
~I’d rather not prefix, as this would break existing implementations again. Users can still prefix if they want with the `sizeProp` in the mixin-options.~

**Comment by e2o** [comment]
Actually, scratch what I said. Let’s update both `size` and `resizeObserver` data-props to have the prefix `$_elementQueryMixin_`. They are private after all and shouldn’t be accessed by the user in the first place. I’ll bump to `v3.0.0` to indicate possible breaking changes for users who did use those properties.

**Comment by e2o** [comment]
Resolved in https://github.com/e2o/vue-element-query/pull/6 and available in [v3.0.0](https://github.com/e2o/vue-element-query/releases/tag/v3.0.0).

**Comment by christophemarois** [comment]
Awesome! I've been able to successfully replace my monkey-patched version of `vue-element-query` by the `3.x` version, and there's no conflict with other libs anymore.

Thanks for the good work!
